src/e2e-test/features/bigquery/source/BigQueryToGCS.feature

```diff
@@ -116,3 +116,39 @@ Feature: BigQuery source - Verification of BigQuery to GCS successful data trans
     Then Open and capture logs
     Then Verify the pipeline status is "Succeeded"
     Then Verify data is transferred to target GCS bucket
+
+  @BQ_SOURCE_DATATYPE_TEST @GCS_CSV_MULTI_PART_UPLOAD
+Scenario:Validate successful records transfer from BigQuery to GCS
```

Comment:
Change the scenario-> Validate successful records transfer from BigQuery to GCS with bucket having delete multi part upload policy enabled

Reply:
Changes Added as per the Suggestion.